test/jdk/java/nio/channels/FileChannel/TransferTo6GBFile.java

```diff
@@ -52,15 +55,12 @@ public void xferTest08() throws Exception { // for bug 6253145
         final long G = 1024L * 1024L * 1024L;
 
         // Create 6GB file
-
-        File file = File.createTempFile("source", null);
-        file.deleteOnExit();
-
-        RandomAccessFile raf = new RandomAccessFile(file, "rw");
-        FileChannel fc = raf.getChannel();
+        Path file = FileChannelUtils.createSparseTempFile("source", null);
+        file.toFile().deleteOnExit();
 
         out.println("  Writing large file...");
         long t0 = System.nanoTime();
+        FileChannel fc = FileChannel.open(file, READ, WRITE);
```

Comment:
You can probably change this to use try-with-resources too and maybe "READ" can be dropped as this loo just writes to the file.